openfisca_france/parameters/tns.xml

```diff
@@ -1,60 +1,60 @@
 <NODE code="tns" description="Travailleurs non-salariés" origin="openfisca">
   <NODE code="auto_entrepreneur" description="Taux global de cotisations sociales et formation à appliquer au CA de l'auto-entrepreneur" origin="openfisca">
     <CODE code="achat_revente" description="Activité de type Achat-Revente" format="percent" origin="openfisca">
-      <VALUE deb="2016-01-01" fuzzy="true" valeur="0.134" />
-      <VALUE deb="2015-01-01" fin="2015-12-31" valeur="0.134" />
-      <VALUE deb="2008-01-01" fin="2014-12-31" valeur="0.142" />
+      <VALUE deb="2016-01-01" valeur="0.134" />
+      <VALUE deb="2015-01-01" valeur="0.134" />
+      <VALUE deb="2008-01-01" valeur="0.142" />
     </CODE>
     <CODE code="bic" description="Activité de type BIC" format="percent" origin="openfisca">
-      <VALUE deb="2016-01-01" fuzzy="true" valeur="0.232" />
-      <VALUE deb="2015-01-01" fin="2015-12-31" valeur="0.232" />
-      <VALUE deb="2008-01-01" fin="2014-12-31" valeur="0.248" />
+      <VALUE deb="2016-01-01" valeur="0.232" />
+      <VALUE deb="2015-01-01" valeur="0.232" />
+      <VALUE deb="2008-01-01" valeur="0.248" />
     </CODE>
     <CODE code="bnc" description="Activité de type BNC" format="percent" origin="openfisca">
-      <VALUE deb="2016-01-01" fuzzy="true" valeur="0.231" />
-      <VALUE deb="2015-01-01" fin="2015-12-31" valeur="0.231" />
-      <VALUE deb="2008-01-01" fin="2014-12-31" valeur="0.235" />
+      <VALUE deb="2016-01-01" valeur="0.231" />
+      <VALUE deb="2015-01-01" valeur="0.231" />
+      <VALUE deb="2008-01-01" valeur="0.235" />
     </CODE>
   </NODE>
   <NODE code="exploitant_agricole" description="Exploitants agricoles non salariés" origin="openfisca">
     <CODE code="maj_1e_2ad" description="Majoration du plafond pour 1er enfant quand deux adultes" origin="openfisca">
-      <VALUE deb="2016-01-01" fuzzy="true" valeur="0.3" />
-      <VALUE deb="2008-01-01" fin="2015-12-31" valeur="0.3" />
+      <VALUE deb="2016-01-01" valeur="0.3" />
+      <VALUE deb="2008-01-01" valeur="0.3" />
     </CODE>
     <CODE code="maj_2p" description="Majoration du plafond pour un foyer de deux personnes" origin="openfisca">
-      <VALUE deb="2016-01-01" fuzzy="true" valeur="0.5" />
-      <VALUE deb="2008-01-01" fin="2015-12-31" valeur="0.5" />
+      <VALUE deb="2016-01-01" valeur="0.5" />
+      <VALUE deb="2008-01-01" valeur="0.5" />
```

Comment:
This is not a lot of work to check in the transformation script if consecutive values are the same or not. However the redundancy was already in the original file. IMHO this check is exceeding the core team's purpose.